FiveCalls/FiveCalls/Dashboard.swift

```diff
@@ -12,24 +12,51 @@ struct Dashboard: View {
     @EnvironmentObject var appState: AppState
 
     @State var showLocationSheet = false
+    @State var showRemindersSheet = false
 
     let op = Operator()
     
     var body: some View {
         NavigationView {
             ScrollView {
                 VStack(alignment: .leading, spacing: 10) {
-                    LocationHeader(location: appState.location, fetchingContacts: appState.fetchingContacts)
-                        .padding(.bottom, 10)
-                        .onTapGesture {
-                            showLocationSheet.toggle()
+                    HStack {
+                        if USE_SHEET {
```

Comment:
I added this variable to switch between using a sheet as before and pushing on navigation stack. We can decide which we want to keep.